doc/modules/ROOT/pages/index.adoc

```diff
@@ -1,46 +1,154 @@
-////
-This is the base template for Hazelcast integration module guides.
+:github-address: https://github.com/hazelcast-guides/kubernetes-wan
+:templates-url: templates:ROOT:page$/
+:hazelcast: Hazelcast IMDG
+:framework: Kubernetes
 
-You can clone this repository, work on it and create your guide, and then push to a new repository.
-////
 
-:github-address: [github-repo-address-of-your-guide]
+= Hazelcast WAN Replication (Enterprise Only)
 
-:templates-url: https://raw.githubusercontent.com/hazelcast-guides/adoc-templates/master
+This a complete example presenting how to set two Hazelcast clusters (deployed in two different Kubernetes environments) with the WAN Replication in between them.
 
-// Use this relative url if you are going to publish the guide on the guides site.
-// Note that this url will not work locally and raise asciidoctor errors.
-// So, complete the guide with the above url and set the below one just before 
-// publishing on the guides site.
-//
-// :templates-url: templates:ROOT:page$/
+include::{templates-url}/link-to-repo.adoc[]
 
-= Title
+== What You’ll Learn
 
-// Content entered directly below the header but before the first section heading is called the preamble.
+In this guide you will create two Hazelcast clusters and will set up WAN Replication between them.
 
-include::{templates-url}/link-to-repo.adoc[]
+== Introduction
 
-== What You’ll Learn
+This example focuses on the WAN Replication feature and assumes that you have some general knowledge about Hazelcast on Kubernetes. Here are some resources:
+
+* https://github.com/hazelcast/hazelcast-kubernetes[Hazelcast Kubernetes README]
+* https://github.com/hazelcast/hazelcast-code-samples/blob/master/hazelcast-integration/kubernetes[Hazelcast Kubernetes Code Sample]
+* https://github.com/hazelcast/hazelcast-code-samples/blob/master/hazelcast-integration/kubernetes/samples/embedded[Hazelcast Kubernetes Embedded Code Sample]
```

Comment:
Could we refer to other kubernetes guides rather than code samples here? Especially for this line:
> The example also assumes you have two running Kubernetes clusters

"... otherwise, you can get started with ... guide" or something like that.

Reply:
Do we have the guide for starting the k8s cluster? The ones that I've all assumes that there is a k8s cluster setup

Reply:
Ah, I was thinking https://guides.hazelcast.org/kubernetes-embedded could be a good start but I see it's based on a spring app. Code samples are good enough then.